.github/workflows/test_tutorial_workflow.yml

```
@@ -0,0 +1,56 @@
+name: test-tutorial-workflow
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      rose_ref:
+        description: The Rose branch to test against
+        required: false
+      cylc_ref:
+        description: The Cylc branch to test against
+        required: false
+      cylc_rose_ref:
+        description: The Cylc branch to test against
+        required: false
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: configure python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+
+      - name: checkout cylc-doc
+        uses: actions/checkout@v2
+
+      - name: install dependencies
+        uses: ./.github/actions/install-dependencies
```

See my earlier comment
```suggestion
      - name: Install docs/tutorial dependencies
        uses: cylc/cylc-doc/.github/actions/install-dependencies@master
```